### ISSUE TITLE
adding a schema on config center is not working

### ISSUE THREAD

**Issue by myzero1** [apache/incubator-shardingsphere#5043]
## Bug Report

**adding a schema on config center is not working,but updating authentication or props is working,before rebooting sharding-proxy. adding a schema on config center is the same as  [#2346](https://github.com/apache/incubator-shardingsphere/issues/2346)**

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/incubator-shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response **more than 7 days** and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?

- [x] 4.0.0
- [x] 4.0.1

### Which project did you use? Sharding-JDBC or Sharding-Proxy?
Sharding-Proxy
### Expected behavior
After adding a schema on config center, I hope the Sharding Rroxy could refresh and load configuration automatically or semi-automatic(by sending a command manually), that is, Then I can see that new schema by `show databases;` **adding a schema on config center is the same as  [#2346](https://github.com/apache/incubator-shardingsphere/issues/2346)**
### Actual behavior
![](https://raw.githubusercontent.com/myzero1/show-time/master/tmp/QQ%E5%9B%BE%E7%89%8720200331175222.png)
![](https://raw.githubusercontent.com/myzero1/show-time/master/tmp/QQ%E5%9B%BE%E7%89%8720200331175232.png)
![](https://raw.githubusercontent.com/myzero1/show-time/master/tmp/QQ%E5%9B%BE%E7%89%8720200331175941.png)

**Comment by kimmking** [comment]
I will follow this issue.

**Comment by tristaZero** [comment]
Assigned.

**Comment by kimmking** [comment]
Until now, sharding-proxy's schemas are base on the config during bootstrap, and then we can't add a new one or remove a existing one.
This behavior lead to that proxyA and proxyB both in the same namespace and orchestrationName, but have differect logic schemas, for examples, proxyA has schema1&schema2, proxyB has schema1&schema2&schema3.

As the design principles, several proxy nodes in the same namespace and orchestrationName, combine a cluster group. They should be the same for each other.

Then we need to modify this module to make sure:

- [ ] Add&Remove schemas in sharding-ui
- [ ] Sync all schemas changed to proxy nodes
- [ ] Process sharding-jdbc logic_db with this case

It's a big issue and a new feature, @menghaoranss will follow it too.

**Comment by menghaoranss** [comment]
@kimmking @myzero1 
I tracked the source code and found that the listener event of updating the schema node content has been supported in version 4.1.0. Refer to PR [#4260](https://github.com/apache/incubator-shardingsphere/pull/4260). When adding or deleting a schema directly through zk client, the proxy can receive the event and reload the schemas, but the modification is missing during UI operation causes zk to fail to trigger listening. I have submitted a PR to fix this problem, and we should discuss remove schemas in sharding-ui and the problem of  sharding-jdbc logic_schema later.

![微信图片_20200403084338](https://user-images.githubusercontent.com/60651792/78312791-81fb8280-7587-11ea-935f-ee1c09192f48.png)
